WidgetInspector.cpp

```diff
@@ -115,6 +120,35 @@ void WidgetInspector::copyDebuggerReference()
 #endif
 }
 
+void WidgetInspector::flashWidget()
+{
+	ObjectProxy::Pointer object = m_objectInspector->object();
+	if (!object)
+	{
+		return;
+	}
+
+	QString origStyle;
+	for (const auto &property : object->properties())
+	{
+		if (property.name == "styleSheet")
```

Comment:
If the widget is in the middle of an earlier flash sequence when the button is clicked, this loop may capture the flashed stylesheet. The result is that the flash effect remains on at the end of the sequence.